cmd/dep/status_test.go

```diff
@@ -368,7 +369,7 @@ func TestCollectConstraints(t *testing.T) {
 			wantConstraints: constraintsCollection{},
 		},
 		{
-			name: "with multiple constraints",
+			name: "with multiple constraints from dependencies",
```

Comment:
"from lock" would be the better term, because those are actually from the lock file. Dependencies could be a little confusing.